doc/tutorials/Chapter_1/01-Setup_Installation.md

```diff
@@ -0,0 +1,308 @@
+        If(en, then: [val < nextVal])
+      ])
+    ]);
+  }
+}
+
+// Let's simulate with this counter a little, generate a waveform, and take a
+// look at generated SystemVerilog.
+Future<void> main({bool noPrint = false}) async {
+  // Define some local signals.
+  final en = Logic(name: 'en');
+  final reset = Logic(name: 'reset');
+
+  // Generate a simple clock.  This will run along by itself as
+  // the Simulator goes.
+  final clk = SimpleClockGenerator(10).clk;
+
+  // Build a counter.
+  final counter = Counter(en, reset, clk);
+
+  // Before we can simulate or generate code with the counter, we need
+  // to build it.
+  await counter.build();
+
+  // Let's see what this module looks like as SystemVerilog, so we can pass it
+  // to other tools.
+  final systemVerilogCode = counter.generateSynth();
+  if (!noPrint) {
+    print(systemVerilogCode);
+  }
+
+  // Now let's try simulating!
+
+  // Let's start off with a disabled counter and asserting reset.
+  en.inject(0);
+  reset.inject(1);
+
+  // Attach a waveform dumper so we can see what happens.
+  if (!noPrint) {
+    WaveDumper(counter);
+  }
+
+  // Drop reset at time 25.
+  Simulator.registerAction(25, () => reset.put(0));
+
+  // Raise enable at time 45.
+  Simulator.registerAction(45, () => en.put(1));
+
+  // Print a message when we're done with the simulation!
+  Simulator.registerAction(100, () {
+    if (!noPrint) {
+      print('Simulation completed!');
+    }
+  });
+
+  // Set a maximum time for the simulation so it doesn't keep running forever.
+  Simulator.setMaxSimTime(100);
+
+  // Kick off the simulation.
+  await Simulator.run();
+
+  // We can take a look at the waves now.
+  if (!noPrint) {
+    print('To view waves, check out waves.vcd with a waveform viewer'
+        ' (e.g. `gtkwave waves.vcd`).');
+  }
+}
+```
+
+Finally, run the main function and observe if you can see the SystemVerilog code generated by running the command:
+
+```shell
+dart run
+```
+
+Well done! Your setup has been completed successfully. The successful generation of SystemVerilog code confirms that your configuration is in good order.
+
+
+### Option 2: Install from Source
+
+Clone ROHD repository to the local directory. (Install From Source). On your terminal, run 
+
+```shell
+cd C:\
+git clone https://github.com/intel/rohd.git
+```
+
+Next, open up your repository in VSCode using the command:
+
+```shell
+code rohd
+```
+
+You will see VSCode automatically open up your ROHD folder. 
+
+![step 2](assets/localSetup/step2.PNG)
+
+Open up terminal in your VSCode by go to view -> terminal. Then, get the rohd package downloaded using the command below.
+
+```cmd
+dart pub get
+```
+Then, run Rohd example.
+
+Open up `example` folder on the left navigation panel and click on `example.dart` to bring forward the first example of ROHD. After that, navigate to the main function at below of line 58 and click on the `Run` at `Run | Debug`.
+
+![step 3](assets/localSetup/step4.PNG)
```

Comment:
The name is `step 3` and the link is `...step4.png`.

Reply:
Let me change it 🤣